config/i18n-tasks.yml

```diff
@@ -0,0 +1,145 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
+
+# The "main" locale.
+base_locale: en
+## All available locales are inferred from the data by default. Alternatively, specify them explicitly:
+# locales: [es, fr]
+## Reporting locale, default: en. Available: en, ru.
+# internal_locale: en
+
+# Read and write translations.
+data:
+  ## Translations are read from the file system. Supported format: YAML, JSON.
+  ## Provide a custom adapter:
+  # adapter: I18n::Tasks::Data::FileSystem
+
+  # Locale files or `File.find` patterns where translations are read from:
+  read:
+    ## Default:
+    # - config/locales/%{locale}.yml
+    ## More files:
+    - config/locales/%{locale}/*.yml
+
+
+  # Locale files to write new keys to, based on a list of key pattern => file rules. Matched from top to bottom:
+  # `i18n-tasks normalize -p` will force move the keys according to these rules
+  write:
+     - ['{
+            activemodel,
+            activerecord,
+            citizens,
+            contacts,
+            devise,
+            forms,
+            generic,
+            helpers,
+            layouts,
+            model_enum_translations,
+            providers,
+            shared}.*', 'config/locales/%{locale}/\1.yml']
+    ## Catch-all default:
+     - config/locales/%{locale}/default.yml
```

Comment:
Can this be done dynamically? For example this gets all the file names in the current locales:
```ruby
Dir[Rails.root.join('config/locales/*.yml')].map{|p| File.basename(p)}
```
There is an option to use an erb version of `config/i18n-tasks.yml` so it looks like you can mix ruby into the YAML: `config/i18n-tasks.yml.erb`